utils/encoding.go

```diff
@@ -5,6 +5,11 @@ import (
 	"strings"
 )
 
+// Decode a hex string. Hex string can be optionally prefixed with 0x.
+func HexEncode(input []byte) string {
+	return "0x" + hex.EncodeToString(input)
+}
+
```

Comment:
what do you mean by optional? this always adds 0x

Reply:
copy/paste error on the comment, fixed